arch/lkl/defconfig

```diff
@@ -32,8 +32,8 @@ CONFIG_EXT4_FS_SECURITY=y
 CONFIG_XFS_FS=y
 CONFIG_XFS_POSIX_ACL=y
 CONFIG_ISO9660_FS=y
-CONFIG_BTRFS_FS=y
-CONFIG_BTRFS_FS_POSIX_ACL=y
+#CONFIG_BTRFS_FS=y
```

Comment:
I was not aware (and not really awake while writing that answer). Thanks for the explanation.